src/Microsoft.Azure.WebJobs.Extension.Storage/Blobs/BlobClient.cs

```diff
@@ -4,24 +4,25 @@
 using System;
 using System.Globalization;
 using System.Text.RegularExpressions;
-using Microsoft.Azure.WebJobs.Host.Storage.Blob;
 using Microsoft.Azure.WebJobs.Description;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
 
-namespace Microsoft.Azure.WebJobs.Host.Blobs
+namespace Microsoft.Azure.WebJobs
 {
     internal static class BlobClient
     {
         // Tested against storage service on July 2016. All other unsafe and reserved characters work fine.
         private static readonly char[] UnsafeBlobNameCharacters = { '\\' };
 
-        public static string GetAccountName(IStorageBlobClient client)
+        public static string GetAccountName(this CloudStorageAccount client)
```

Comment:
This param name no longer matches the type

Reply:
Done.